modules/runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/config/ConfigurationGenerator.java

```diff
@@ -392,6 +395,7 @@ protected boolean isGenerationOnce() {
     /**
      * @see #PARAM_FORCE_GENERATION
      */
+    @Deprecated(since = "11.1") // not used
```

Comment:
11.5 ?

Reply:
Removed from appropriate commit as it is an intermediate thing.